dbflow/src/main/java/com/raizlabs/android/dbflow/sql/queriable/StringQuery.java

```diff
@@ -44,7 +64,41 @@ public Cursor query() {
 
     @Override
     public Cursor query(DatabaseWrapper databaseWrapper) {
-        return databaseWrapper.rawQuery(query, null);
+        String[] selectionArgs = new String[params.size()];
+
+        for (ParamsHolder holder : params) {
```

Comment:
#? why not just bind values directly to the list as a `List<String` then call `toArray()` on that list passed into the `rawQuery()` method?